internal/service/sagemaker/feature_group.go

```diff
@@ -78,7 +76,6 @@ func ResourceFeatureGroup() *schema.Resource {
 						"feature_type": {
 							Type:         schema.TypeString,
 							Optional:     true,
-							ForceNew:     true,
 							ValidateFunc: validation.StringInSlice(sagemaker.FeatureType_Values(), false),
```

Comment:
Hi @nikhil-goenka There is no update function defined inside feature group update to allow this.
You would need add an update block similar to - https://github.com/hashicorp/terraform-provider-aws/blob/c9f891ae25e6914908996da01ba13d7c1a9482fc/internal/service/sagemaker/feature_group.go#L259

Go SDK https://docs.aws.amazon.com/sdk-for-go/api/service/sagemaker/#UpdateFeatureGroupInput